libqtile/core/manager.py

```diff
@@ -382,6 +390,12 @@ def _process_screens(self, reloading: bool = False) -> None:
 
         self.screens = screens
 
+    def remove_dbus_rules(self) -> None:
+        # Disconnecting the bus connections is enough to remove the match rules.
+        while dbus_bus_connections:
+            bus = dbus_bus_connections.pop()
+            bus.disconnect()
```

Comment:
As this method doesn't use `self`, could we move it into `utils` as a standalone function?

Reply:
Yes